va/validation-authority_test.go

```diff
@@ -184,65 +187,92 @@ func TestSimpleHttp(t *testing.T) {
 	<-waitChan
 
 	finChall, err := va.validateSimpleHTTP(ident, chall)
-	test.AssertEquals(t, finChall.Status, core.StatusValid)
-	test.AssertNotError(t, err, chall.Path)
+	// TODO(Issue #401): Until #401 is resolved ignore DNS timeouts from non-local resolver
```

Comment:
Given that we don't need SimpleHTTP to do its own DNS checking, the changes in this method are unnecessary.